common/config/rush/common-versions.json

```diff
@@ -63,6 +63,6 @@
 
     // HOTFIX branch for identity
     "@azure/core-tracing": ["1.0.0-preview.11"],
-    "@azure/core-http": ["^1.2.4"]
+    "@azure/core-http": ["^1.2.4", "^2.0.0"]
```

Comment:
Not a big deal but I'm surprised these are needed for the hotfix since you depend on core-http 2 and core-tracing preview 12.